### ISSUE TITLE
bvh_header file missing

### ISSUE THREAD

**Issue by hongsukchoi** [rludlow/3d-pose-2d-keypoints#3]
Hi Thank you for your contribution.
I am interested in converting bvh file data to 3d coordinate data and
I wonder where is the bvh_head.txt? you mentioned in README.

**Comment by pratishthavrm** [comment]
if you are successfully done this code 
plz help me for the error:
ModuleNotFoundError: No module named 'draw_poses'

**Comment by sowmyakavali** [comment]
> Hi Thank you for your contribution. I am interested in converting bvh file data to 3d coordinate data and I wonder where is the bvh_head.txt? you mentioned in README.

yes i am also not able to understand it, is it enough that if we create a empty file?
